app/assets/stylesheets/views/_browse.scss

```diff
@@ -92,4 +92,18 @@ $browse-header-background-colour: #263135;
   @include govuk-media-query($from: "tablet") {
     margin-bottom: govuk-spacing(6);
   }
-}
\ No newline at end of file
+}
+
+.browse__govuk-chat-promo {
+  @include govuk-media-query($until: "tablet") {
+    border-top: 4px solid govuk-colour("blue");
+    padding-top: govuk-spacing(5);
+  }
+}
+
+.browse_second_level__govuk-chat-promo {
```

Comment:
This class doesn't seem consistent with GOV.UK BEM as we don't have precedence for `.browse_second_level`

I think the most appropriate thing would be to have this as a modifier. So something like:

```
.browse__govuk-chat-promo {
  @include govuk-media-query($until: "tablet") {
    border-top: 4px solid govuk-colour("blue");
    padding-top: govuk-spacing(5);
  }
}

.browse__govuk-chat-promo--second-level {
  @include govuk-media-query($until: "tablet") {
    border-top-width: 2px;
  }
}
```